### PR TITLE
Added submit button in quiz bottom sheet

### PR DESCRIPTION
**Pull request by thedevelopersanjeev**
Fixes part five of #181 

Changes: Added submit button in quiz bottom sheet

Screenshots for the change:
Screenshot_20190519-035335

On clicking submit button
Screenshot_20190519-035342

### CODE REVIEW

**Comment by thedevelopersanjeev** [comment]
Please review @aggarwalpulkit596 
I didn't understand what to fix in the fragment transitions as it is already using custom animations of slide_in_left and slide_in_right.

**Comment by thedevelopersanjeev** [comment]
Please review this PR @aggarwalpulkit596

**Comment by aggarwalpulkit596** [comment]
th

**Comment by aggarwalpulkit596** [comment]
Why there is white backg in dialog box and add a little bit of bottom margin to submit quiz button and change button it's not looking nice

**Comment by aggarwalpulkit596** [comment]
Add updated image

**Comment by thedevelopersanjeev** [comment]
removed the white background from dialog and increased bottom margin of the submit button.
which button are you talking about which needs to be changed @aggarwalpulkit596 
Submit, Cancel or Yes?

**Comment by aggarwalpulkit596** [comment]
Submit button

**Comment by thedevelopersanjeev** [comment]
here it is @aggarwalpulkit596 
Screenshot_20190519-203208

**Comment by thedevelopersanjeev** [comment]
Can you provide me some design of submit button, how it should look like?
I was trying to keep its style the same as other buttons in the app.

**Comment by aggarwalpulkit596** [comment]
Yes but two identical buttons are not looking good and  dialog is still now completely transparent check other dialogs

**Comment by thedevelopersanjeev** [comment]
@aggarwalpulkit596  please review the new submit button design
and I am not able to spot the difference between this dialog and other dialogs
could you please elaborate your last point about dialog a bit more?
Thanks.
Screenshot_1558303155

**Comment by aggarwalpulkit596** [comment]
@yeah button will work for now and if you see closely the background is not yet transparent there is w white background behind the dialog see around the edges

File: app/src/main/res/layout/custom_confirm_dialog.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (aggarwalpulkit596) [inline app/src/main/res/layout/custom_confirm_dialog.xml]: Infact this is not needed use a previous dialog just change its text

File: app/src/main/java/com/codingblocks/cbonlineapp/fragments/QuizFragment.kt [hunk 2]
```diff
@@ -122,6 +131,48 @@ class QuizFragment : Fragment(), AnkoLogger, ViewPager.OnPageChangeListener, Vie
...
+        customView.description.text = "Are you sure to submit the quiz?"
+        builder.setCancelable(false)
+        builder.setView(customView)
+        val dialog = builder.create()
+        customView.cancelBtn.setOnClickListener {
+            dialog.dismiss()
+        }
+        customView.okBtn.setOnClickListener {
+            Clients.onlineV2JsonApi.sumbitQuizById(quizAttemptId).enqueue(retrofitCallback { throwable, response ->
+                val fragmentManager = fragmentManager!!
```
Comment (aggarwalpulkit596) [inline app/src/main/java/com/codingblocks/cbonlineapp/fragments/QuizFragment.kt]: make a util function for this and use that in the future to handle fragment transactions(this is optional you may do it if you feel like)

File: app/src/main/res/layout/custom_confirm_dialog.xml (hunk 1 above)
Comment (thedevelopersanjeev) [reply app/src/main/res/layout/custom_confirm_dialog.xml]: done

File: app/src/main/java/com/codingblocks/cbonlineapp/fragments/QuizFragment.kt (hunk 2 above)
Comment (thedevelopersanjeev) [reply app/src/main/java/com/codingblocks/cbonlineapp/fragments/QuizFragment.kt]: done

**Comment by aggarwalpulkit596** [comment]
update image 😅

**Comment by thedevelopersanjeev** [comment]
Made the util function for fragment transaction and here is the updated image : )
@aggarwalpulkit596 
scr

**Comment by aggarwalpulkit596** [comment]
Can you also squash your commits into a single commit

**Comment by aggarwalpulkit596** [comment]
@thedevelopersanjeev claim 150 points for this

**Comment by thedevelopersanjeev** [comment]
I am squashing the commits...
I am not able to redeem the points on the boss website. It asks me to connect my GitHub account but I have already connected it.
I have posted the issue on gitter.

**Comment by aggarwalpulkit596** [comment]
@thedevelopersanjeev i will look into it don't worry there might be some issue on the website it'll get fixed till then note down your point along with url of pr and issue

**Comment by thedevelopersanjeev** [comment]
rebase and squash done. @aggarwalpulkit596

**Pull request merged**